handlers.go

```diff
@@ -422,18 +423,19 @@ func (r *oauthProxy) tokenHandler(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 	w.Header().Set("Content-Type", "application/json")
-	w.Write(user.token.Payload)
+	_, _ = w.Write(user.token.Payload)
```

Comment:
What's the difference between `w.Write(user.token.Payload)` and `_, _ = w.Write(user.token.Payload)` ?

Reply:
Again, linting rule for code smells: one should not ignore returned values, unless doing so explicitly. Hence the _, _ to explicitly drop the result.